src/main/java/Duckie.java

```diff
@@ -0,0 +1,47 @@
+package main.java;
+
+
+import main.java.classes.*;
```

Comment:
Should the * here be replaced with explicit import statements? I believe it's part of the coding standard.